### ISSUE TITLE
Error messages

### ISSUE THREAD

**Issue by urtson** [geocollections/sarv-file-upload#25]
A green popup flag appears when API returns with status 200 but it can be an error message that is returned and files are actually not uploaded. API errors should not appear when requests are correct and API is correctly configured but it may happen in some special untested cases. The user should then be aware that something is wrong and files did not upload.
Right now red popup with 'something went wrong' appears only in case of apache or django errors which again should not emerge if API was ok.
It is enough to show the same red popup in case of API error without showing the actual message itself.
Right now messages are returned as strings, should it be instead in json like {"error":message} to catch them properly?

**Comment by enmust** [comment]
Yes error messages should be easily noticeable and I think {"error": "errorMessage"} is okay.

**Comment by urtson** [comment]
Messages are now returned in json
in case of error 'error':
In case of success 'message':'Added attachment/imageset $file name/imageset ID' that can be displayed on green OK flag

**Comment by enmust** [comment]
Done
